src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,22 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface MoviesListProps {
+  movies: {
+    title: string;
+    description: string;
+    imdbUrl: string;
+    imgUrl: string;
+    imdbId: string;
+  } [];
```

Comment:
Same. BTW you already have it in Movie.ts
```suggestion
  movies: Movie[];
```